EstateAdministrationUI.IntegrationTests/Common/Setup.cs

```diff
@@ -0,0 +1,61 @@
+﻿namespace EstateAdministrationUI.IntegrationTests.Common
+{
+    using System;
+    using Ductus.FluentDocker.Services;
+    using Ductus.FluentDocker.Services.Extensions;
+    using NLog;
+    using Shared.Logger;
+    using Shouldly;
+    using TechTalk.SpecFlow;
+
+    [Binding]
+    public class Setup
+    {
+        public static IContainerService DatabaseServerContainer;
+        private static String DbConnectionStringWithNoDatabase;
+        public static INetworkService DatabaseServerNetwork;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=5485852)